docs/contributing.rst

```diff
@@ -204,7 +204,7 @@ A checklist for creating, testing, and distributing a new version.
 #. Add a ``:release:`` line to :file:`docs/changelog.rst`.
 #. Commit and open a pull request for review.
 #. Merge the pull request, and ensure the `Travis`_ build passes.
-#. Create a new git tag with ``git tag -m tag {version}``.
+#. Create a new git tag with ``git tag -m "Release v{version}" {version}``.
 #. Push the new tag with ``git push upstream {version}``.
 #. Watch the release in `Travis`_.
 #. Send announcement email to `distutils-sig mailing list`_ and celebrate.
```

Comment:
Not urgent, but is it still necessary to announce on distutils-sig? I don't see any recent announcements for twine, and very few release announcements in general. @brainwane?

Reply:
@bhrutledge Your phrasing makes it hard for me to answer the question.

"Necessary": following the code of conduct and keeping promises
"Good idea": lots of things, which, in my opinion, include announcing on distutils-sig, since some people (like @`techalchemy) follow that but have a hard time with Discourse
"Eh, sure, why not": various things
"Not worth bothering": various things
"No, don't do this": various things

I would guess that, on the question of "should a maintainer announce every Twine release on distutils-sig?" your colleagues range from "not worth bothering" to "good idea". When I make a release I generally announce it on distutils-sig. I am in favor of it staying in the contributor guide, but if someone takes it out, I will not rend my garments nor swear revenge on anyone's household.